processing/src/main/java/io/druid/segment/incremental/IncrementalIndex.java

```diff
@@ -268,6 +270,7 @@ public int lookupId(String name)
   private final Map<String, MetricDesc> metricDescs;
   private final Map<String, DimensionDesc> dimensionDescs;
   private final Map<String, ColumnCapabilitiesImpl> columnCapabilities;
+  private final DimensionValueHolder dimValues;
```

Comment:
Almost two months had passed from the first patch and things are changed much in hereabout. I'll check if I can remove more codes in here.